fix headers in PhysicsTools/PatAlgos

mrodozov opened the pull request:
Fix headers related to #15248

Commit:
fix headers in PhysicsTools/PatAlgos

mrodozov commented:
please test

cmsbuild commented:
+code-checks

Logs: https://cmssdt.cern.ch/SDT/code-checks/cms-sw-PR-23370/4879

cmsbuild commented:
The tests are being triggered in jenkins.

perrotta commented on PhysicsTools/PatAlgos/plugins/BaseMVAValueMapProducer.h:
```diff
@@ -40,6 +40,9 @@
 #include "CommonTools/Utils/interface/StringObjectFunction.h"
 #include "DataFormats/Common/interface/ValueMap.h"
 #include "CommonTools/Utils/interface/TMVAZipReader.h"
+#include "DataFormats/PatCandidates/interface/Jet.h"
```
Can't these three includes get now removed from PhysicsTools/PatAlgos/plugins/BaseMVAValueMapProducer.cc, then?

perrotta commented:
@mrodozov : please let us know if you plan to clean up the implementation file as suggested in https://github.com/cms-sw/cmssw/pull/23370#pullrequestreview-124279147, or if you prefere to let things as they are now instead

Commit:
Update BaseMVAValueMapProducer.cc

Update on this change request: https://github.com/cms-sw/cmssw/pull/23370#pullrequestreview-124279147

cmsbuild commented:
The code-checks are being triggered in jenkins.

mrodozov commented:
please test

mrodozov commented:
@perrotta updated

cmsbuild commented:
+code-checks

Logs: https://cmssdt.cern.ch/SDT/code-checks/cms-sw-PR-23370/5033

cmsbuild commented:
The tests are being triggered in jenkins.

cmsbuild commented:
Pull request #23370 was updated. @perrotta, @monttj, @cmsbuild, @slava77, @gpetruc, @arizzi can you please check and sign again.

perrotta commented:
Thank you Mircho!
Then, I won't be shy and I'll ask the same also for #23463 (as well as  
from previous #23385)


Mircho Rodozov <notifications@github.com> ha scritto:

> @perrotta updated
>
> --
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub:
> https://github.com/cms-sw/cmssw/pull/23370#issuecomment-394678863

cmsbuild commented:
+1
Tested at: 3ff1dd4a7931b9b05ccca6809be3389f1bab2aaf
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-23370/28476/summary.html

cmsbuild commented:
Comparison job queued.

cmsbuild commented:
Comparison is ready
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-23370/28476/summary.html

Comparison Summary:
 - No significant changes to the logs found
 - Reco comparison results: 0 differences found in the comparisons
 - DQMHistoTests: Total files compared: 31
 - DQMHistoTests: Total histograms compared: 2902471
 - DQMHistoTests: Total failures: 2
 - DQMHistoTests: Total nulls: 0
 - DQMHistoTests: Total successes: 2902279
 - DQMHistoTests: Total skipped: 190
 - DQMHistoTests: Total Missing objects: 0
 - DQMHistoSizes: Histogram memory added: 0.0 KiB( 30 files compared)
 - Checked 128 log files, 14 edm output root files, 31 DQM output files

perrotta commented:
+1
- Technical
- Jenkins tests pass

fabiocos commented:
+1

fabiocos commented:
merge
